aries-site/src/pages/design-tokens/color-pairing.mdx

```diff
@@ -0,0 +1,66 @@
+import { Box, Grid, Notification, Text } from 'grommet';
+import { Trigger } from 'grommet-icons';
+import { ContentPane, Example } from '../../layouts';
+import { ColorPreview } from '../../components';
+import {
+  SpecialtyBackgrounds,
+  StandardBackgrounds,
+  StandardForegrounds,
+  StrongBackgrounds,
+} from '../../examples';
+
+## Using background and foreground colors
+
+Backgrounds pair with text and icon colors, unless they have a specific color pairing. For example, `color.background.default` or `color.background.front` will work with `color.text.default` or `color.icon.strong`.
+
+## Accessible color pairings
+
+These background colors can be paired with the following text and icon colors and will meet accessibility requirements for color contrast.
+
+Combine background and text and icon colors with confidence as all contrast ratios have been tested.
+
+<StandardBackgrounds />
```

Comment:
Question -- are we expecting these to be an exhaustive list? or just some examples to get the concept across?

Just wondering in terms of long-term maintenance for how these examples are built up. Right now, they're hard coded.

cc: @halocline